plugin/acl/README.md

```diff
@@ -0,0 +1,98 @@
+# acl
+
+*acl* - enforces basic access control policies and prevents unauthorized access to protected servers.
+
+## Description
+
+With `acl` enabled, users are able to filter DNS queries with filtering rules, i.e. allowing authorized queries to recurse or blocking unauthorized queries towards protected DNS zones.
+
+This plugin can be used multiple times per Server Block.
+
+## Syntax
+
+```
+acl [ZONES…] {
+    ACTION type QTYPE net SOURCE
```

Comment:
`type` is needed for a future extension?

Reply:
If so, this should expanded upon in the documenation; that future iteration might work on TSIG? How would things works if you want TSIG and IP source addresses? Do we want to support that?

Reply:
Well, `type` and `net` here are simply preserved identifiers to separate different type of tokens. The original idea of this plugin is to filter DNS queries by record types and source IP addresses, but free to refactor and make it more extensible.